Sming/SmingCore/Network/TcpConnection.cpp

```diff
@@ -165,7 +165,8 @@ void TcpConnection::onError(err_t err)
 #ifdef ENABLE_SSL
 	if(ssl) {
 		sslConnected = false;
-		ssl_free(ssl); // ssl_free frees internally also the SSL context and the SSL extension data
+		ssl_free(ssl);
```

Comment:
@ericogr Please, delete that line. The line below should take care to do the right free-ing.